.github/workflows/ci.yml

```
@@ -1,5 +1,5 @@
 name: Run MATLAB Tests on GitHub-Hosted Runner
-on: [push]
+on: pus
```

I think you have a typo here.

```suggestion
on: push
```